### PR TITLE
Attach/Detach scene traversal

### PR DESCRIPTION
**Pull request by benaadams**
As suggested in #6433

### CODE REVIEW

File: src/extras/SceneUtils.js [hunk 1]
```diff
@@ -18,21 +18,46 @@ THREE.SceneUtils = {
...
+
+		var topParent = child.parent;
+		
+		while ( topParent.parent !== undefined ) {
+			
+			topParent = topParent.parent;
+			
+		}
+		
+		topParent.updateMatrixWorld( true );
```
Comment (gero3) [inline src/extras/SceneUtils.js]: This updates the complete scene. Wouldn't it be better if we had a method that calculates the matrixWorld for only 1 parent chain??

Comment (benaadams) [inline src/extras/SceneUtils.js]: Yes, @WestLangley just pointed this out :(

Sensible to drop parent as parameter though? Else you may be trying to remove from an unrelated item

**Comment by benaadams** [comment]
Needs changes...

File: src/extras/SceneUtils.js (hunk 1 above)
Comment (gero3) [inline src/extras/SceneUtils.js]: I agree.

**Pull request closed without merging**